projectDocs/issues/githubIssueTemplateExplanationAndExamples.md

```diff
@@ -10,9 +10,13 @@ If you are not familiar, please take some time to learn about
 **Warning**: In all but exceptional circumstances we require this template to be completed.
 Your issue will likely be closed if this template has not been followed.
 
-We currently have two templates, both are described on this page:
+We currently have four templates:
 - for bug reports
-- for feature requests.
+- for feature requests
+- for special case issues that cannot be easily categorised as either a bug report or a feature request

```

Comment:
the rest of this document outlines how to fill out each of these templates in more detail. It is probably worth adding content for the new issue template. Historically we have tried to minimize the use of inline comments to explain fields in favour of putting the information in this document instead.